packages/server/graphql/mutations/createGitHubTaskIntegration.ts

```diff
@@ -68,44 +71,62 @@ export default {
       })
     }
 
-    // RESOLUTION
     const {content: rawContentStr, meetingId} = task
-    const viewerAuth = await dataLoader.get('githubAuth').load({teamId, userId: viewerId})
-
-    if (!viewerAuth) {
+    const [viewerAuth, assigneeAuth, team, teamMembers] = await Promise.all([
+      dataLoader.get('githubAuth').load({teamId, userId: viewerId}),
+      userId && dataLoader.get('githubAuth').load({teamId, userId}),
```

Comment:
if you use a ternary here you'll get strong types returned from Promise.all, otherwise everything is `unknown`

```ts
userId ? dataLoader(...) : null
```

Reply:
That's a good tip!